src/main/generated/resources/assets/minecolonies/lang/default.json

```diff
@@ -88,7 +88,7 @@
   "com.minecolonies.research.civilian.diligent.name": "Diligent",
   "com.minecolonies.research.civilian.studious.name": "Studious",
   "com.minecolonies.research.civilian.scholarly.name": "Scholarly",
-  "com.minecolonies.research.civilian.scholarly.subtitle": "Homework for the next decade... check!",
+  "com.minecolonies.research.civilian.scholarly.subtitle": "Homework for the next decade...check!",
```

Comment:
I think this looks better with the space, although it's not super important.

Reply:
I have the opposite opinion (and actually removed the space myself in a recent, merged PR), but oh well